diff --git a/src/cmd/gofmt/gofmt.go b/src/cmd/gofmt/gofmt.go
index ce9c299dc8..a6608121a1 100644
--- a/src/cmd/gofmt/gofmt.go
+++ b/src/cmd/gofmt/gofmt.go
@@ -56,6 +56,9 @@ func report(err error) {
 func usage() {
 	fmt.Fprintf(os.Stderr, "usage: gofmt [flags] [path ...]\n")
 	flag.PrintDefaults()
+	fmt.Fprintln(os.Stderr)
+	fmt.Fprintln(os.Stderr, "Is this the gofmt you're looking for?")
+	fmt.Fprintln(os.Stderr, "This gofmt is from github.com/cockroachdb/gostdlib.")
 }

 func initParserMode() {

diff --git a/src/x/tools/cmd/goimports/goimports.go b/src/x/tools/cmd/goimports/goimports.go
index 53235d459c..57681467c7 100644
--- a/src/x/tools/cmd/goimports/goimports.go
+++ b/src/x/tools/cmd/goimports/goimports.go
@@ -58,6 +58,9 @@ func report(err error) {
 func usage() {
 	fmt.Fprintf(os.Stderr, "usage: goimports [flags] [path ...]\n")
 	flag.PrintDefaults()
+	fmt.Fprintln(os.Stderr)
+	fmt.Fprintln(os.Stderr, "Is this the goimports you're looking for?")
+	fmt.Fprintln(os.Stderr, "This goimports is from github.com/cockroachdb/gostdlib.")
 	os.Exit(2)
 }

